### PR TITLE
[New Action] Shopify addTags mutation

### PR DESCRIPTION
**Pull request by DanPete**
Allows users to add tags to Shopify Products, Customers, Orders, Draft Orders, or Articles.

This is my first PR with Pipedream, so I am open to feedback or other advice. Thanks! 🙂
 
CleanShot 2022-07-13 at 21 38 02@2x

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/3435"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

File: components/shopify/actions/add-tags/add-tags.mjs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    `;
+
+    const variables = {
+      id: gid,
+      tags: tags,
+    };
+
+    const res = await this.shopify
+      .getShopifyInstance()
+      .graphql(mutation, variables);
```
Comment (feyzullah) [inline components/shopify/actions/add-tags/add-tags.mjs:71]: Can you please move this call to the  [app file](https://github.com/PipedreamHQ/pipedream/blob/master/components/shopify/shopify.app.mjs)? Like it is done in [`create-customer`](https://github.com/PipedreamHQ/pipedream/blob/master/components/shopify/actions/create-customer/create-customer.mjs)?

Comment (feyzullah) [inline components/shopify/actions/add-tags/add-tags.mjs:71]: If possible, can you please use [_makeGraphQlRequest](https://github.com/PipedreamHQ/pipedream/blob/495e324420b86b831e438521dac8028473db6241/components/shopify/shopify.app.mjs#L433) after moving the call to the [app file](https://github.com/PipedreamHQ/pipedream/blob/master/components/shopify/shopify.app.mjs)?

Comment (DanPete) [reply components/shopify/actions/add-tags/add-tags.mjs]: @feyzullah Thanks for the feed back. I went ahead and moved it to the app file. Let me know if that's what you were thinking.

Comment (DanPete) [reply components/shopify/actions/add-tags/add-tags.mjs]: @feyzullah With the move to the app file, I am now using `_makeGraphQlRequest`. Thanks for pointing me to that function didn't realize it was already there. 🙂

**Comment by feyzullah** [comment]
Hi @DanPete thank you very much for the updates. These are what exactly I meant.
There are very small linter errors and I am going to fix them and push in a minute.
After this, I will be approving and there will be a QA review, so please don't merge before QA guys check this.
Thank you very much, great job!

**Review by feyzullah — APPROVED** [review]
LGTM! Thank you very much @DanPete

**Comment by DanPete** [comment]
> Hi @DanPete thank you very much for the updates. These are what exactly I meant. There are very small linter errors and I am going to fix them and push in a minute. After this, I will be approving and there will be a QA review, so please don't merge before QA guys check this. Thank you very much, great job!

Great, thank you!

Do I need to setup something in my vscode to make sure it is applying the correct formatting?

Right now I have it set to this: 

```
"editor.defaultFormatter": "esbenp.prettier-vscode",
```
and
```
"[javascript]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
  ```
  
  But that seems to be not formatting by Pipedreams `.eslintrc`

**Comment by DanPete** [comment]
@feyzullah Great, I'll try that next time, thanks!

**Comment by vunguyenhung** [comment]
I will merge this PR since @DanPete doesn't have write permission.

**Comment by DanPete** [comment]
Thanks @vunguyenhung! 🙂

**Pull request merged**